policies/high_level_policy.py

```diff
@@ -6,6 +6,64 @@
 from algorithms.dijkstra import dijkstra
 from policies.policy import Policy
 
+def nextSteps(path, nearby):
+    # passes is the number of steps
+    passes=0
+    # basic array for return; these tuples are replaced with directions
+    directionReturn=[[0,0], [0,0]]
+    # boolean indicating if the first step is in x direction or not
+    isX = True
+    # if no change in x, moving in y, isX is False
+    if (len(path)>=3):
+        if (np.subtract(path[2],path[1])[0]==0):
+            isX=False
+    else:  
+        if (nearby):
+            directionReturn[0] = np.flip(np.subtract(path[1],path[0]))
+        return directionReturn
+    # starts at index 1
+    index=1
+    # Only looks at the next two steps
+    while(passes<2):
+        # If at the end of the provided steps, break
+        if(index+1==len(path)):
+            break
+        # initial "step"; will be changed appropriately and placed into return array
+        numSteps=[0,0]
+        # if in the y-direction
+        if (not isX):
```

Comment:
You can simplify this conditional while loop logic:
```suggestion
        check_index = int(isX)
        while (np.subtract(path[index], path[index+1])[check_index] == 0):
```
This will prevent you from having to use two while loops.